wagtail/admin/panels.py

```diff
@@ -99,6 +99,46 @@ def extract_panel_definitions_from_model_class(model, exclude=None):
     return panels
 
 
+def merge_form_options(*options_list):
```

Comment:
We can also make the parameter a list instead of spreading it as arbitrary args. Happy to make the change if that's desirable.